__tests__/app.test.js

```diff
@@ -113,3 +113,52 @@ describe.only("GET /api/articles", () => {
       });
   });
 });
+
+describe.only("GET /api/articles/:article_id/comments", () => {
+  test("200: Returns an array of comments objects", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("200: Returns an array of comments objects with comment_id in ASC order", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
```

Comment:
We need to make sure the comments are in order of **date** created not the id's (which only keep track of the order of insertion which although may seem similar is not the same thing). Use [jest-sorted](https://www.npmjs.com/package/jest-sorted) here would be my recommendation and use the more appropriate property on each comment to do this.